File: param-manager-web/src/main/webapp/js/service/brandService.js
//service level
app.service("brandService",function($http){
    this.findPage = function(page,size,entity){
        return $http.post("../brand/search.do?pageNum="+page+"&pageSize="+size,entity);
    };
    this.findOne = function(id){
        return $http.get("../brand/findOne.do?id=" + id);
    } ;
    this.dele = function(ids){
        return $http.get("../brand/delete.do?ids="+ids);
    }
    this.edit = function(entity){
        return $http.post("../brand/edit.do",entity);
    }
    this.save = function(entity){
        return $http.post("../brand/save.do",entity);
    }

});

File: param-sellerproducts-interface/src/main/java/com/param/sellerproducts/service/BrandService.java
package com.param.sellerproducts.service;

import com.param.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

public interface BrandService {
    List<TbBrand> findAll();

    PageResult findPage(int pageNum, int pageSize);

    void addBrand(TbBrand tbBrand);

    TbBrand findOne(long id);

    void editBrand(TbBrand tbBrand);

    void deleteBrands(long[] ids);

    PageResult findPage(TbBrand tbBrand, int pageNum, int pageSize);

    List<Map> findBrandOptions();
}
